rpmconf/Install/zmsetup.pl

```diff
@@ -3664,14 +3664,14 @@ sub createOnlyofficeMenu {
     #    set the server host name at server level config
     if (isEnabled("zimbra-store")) {
       $config{ONLYOFFICESTANDALONE} = "no";
-      $config{ONLYOFFICEHOSTNAME} = $config{HOSTNAME};
     } else {
       $config{ONLYOFFICESTANDALONE} = "yes";
-      if ($config{ONLYOFFICEHOSTNAME} eq "") {
-        $config{ONLYOFFICEHOSTNAME} = $config{HOSTNAME};
-      }
+    }
 
+    if ($config{ONLYOFFICEHOSTNAME} eq "") {
+        $config{ONLYOFFICEHOSTNAME} = $config{HOSTNAME};
```

Comment:
is this `$config{ONLYOFFICEHOSTNAME}` pre initialized some where?

Reply:
No

Reply:
so why are we checking if its empty? or is it getting input during installation?

Reply:
This is a dictionary. If the dictionary doesn't contain a key-value, it will be blank. Also to check if someone has intentionally put blank somehow.

Reply:
ok